kogito-kafka-quickstart-quarkus/README.md

```diff
@@ -65,8 +116,14 @@ To run the generated native executable, generated in `target/`, execute
 
 ### Use the application
 
-To make use of this application it is as simple as putting a message on `travellers` topic with following content (cloud event format) 
+To make use of this application it is as simple as putting a message on `travellers` topic with following content 
+
+* Send Message to Topic
+```
+kafka-console-producer.sh --broker-list localhost:9092 --topic travellers
+```
 
+Content (cloud event format)
 ```
 {
```

Comment:
Please put the json as a one-line, as the user may want to copy/paste it

Reply:
Done @radtriste